data/other/ldap_scheme.yml

```diff
@@ -28,6 +28,12 @@ parents:
             - organizationalUnit
             - top
 
+    ou=aliases:
```

Comment:
you're right. Yesterday I read _ensure_ldap_ou_is_created function too quickly. thanks